lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -145,10 +158,7 @@ public function bindParam($column, &$variable, $type = null, $length = null)
         $column = isset($this->_paramMap[$column]) ? $this->_paramMap[$column] : $column;
 
         if ($type == \PDO::PARAM_LOB) {
-            $lob = oci_new_descriptor($this->_dbh, OCI_D_LOB);
-            $lob->writeTemporary($variable, OCI_TEMP_BLOB);
-
```

Comment:
If I try to write temporary from the reference, it'll fail with the following: `ORA-22275: invalid LOB locator specified`.

WDYT about throwing an error if the `$variable` isn't an OCI-Lob instance?